plugin/pkg/admission/resourcequota/config.go

```diff
@@ -63,5 +79,7 @@ func LoadConfiguration(config io.Reader) (*resourcequotaapi.Configuration, error
 	if !ok {
 		return nil, fmt.Errorf("unexpected type: %T", decodedObj)
 	}
+	// append default quota admission config
+	resourceQuotaConfiguration.LimitedResources = append(resourceQuotaConfiguration.LimitedResources, defaultLimitedResources()...)
```

Comment:
ux on this is weird.  Is it a default or is `requiredLimitedResources`

Reply:
So, if the user provides some `limitedResources` this value will be appended. It's not a default.